data-es7.js

```diff
@@ -1190,6 +1190,59 @@ exports.tests = [
   res: {
     babel:       true,
   }
+},
+{
+  name: 'Observable',
+  category: 'proposal',
+  significance: 'large',
+  link: 'https://github.com/zenparsing/es-observable',
+  'subtests': {
+    'basic support': {
+      exec: function () {/*
+        return typeof Observable !== 'undefined';
+      */}
+    },
+    'Symbol.observable well known symbol': {
+      exec: function () {/*
+        return typeof Symbol.observable === 'symbol';
+      */}
+    },
+    'Observable.prototype.subscribe': {
+      exec: function () {/*
+        return 'subscribe' in Observable.prototype;
```

Comment:
Yes they probably should - I'll add a function check as well as check the return type.